benches/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "benches"
+edition = "2024"
+license.workspace = true
+version.workspace = true
+
+[[bin]]
+name = "streaming"
+path = "src/streaming.rs"
+
+[dependencies]
+anyhow.workspace = true
+clap = { version = "4.5.49", features = ["derive"] }
+futures.workspace = true
+indicatif = "0.18.0"
```

Comment:
since we're using in multiple places, we should extract any common deps into the workspace folder, to avoid any possible dups